ignite/metrics/accumulation.py

```diff
@@ -125,15 +130,20 @@ class Average(VariableAccumulation):
     def __init__(
         self, output_transform: Callable = lambda x: x, device: Union[str, torch.device] = torch.device("cpu")
     ):
-        def _mean_op(a, x):
-            if isinstance(x, torch.Tensor) and x.ndim > 1:
+        def _mean_op(a: Union[float, torch.Tensor], x: Union[float, torch.Tensor]) -> Union[float, torch.Tensor]:
+            if isinstance(x, torch.Tensor) and x.ndim > 1:  # type: ignore[attr-defined]
                 x = x.sum(dim=0)
             return a + x
 
         super(Average, self).__init__(op=_mean_op, output_transform=output_transform, device=device)
 
     @sync_all_reduce("accumulator", "num_examples")
     def compute(self) -> Union[Any, torch.Tensor, numbers.Number]:
+        if self.num_examples is None:
```

Comment:
I'd love to find a way to remove those mypy-happy checks :)

EDIT: Let's do the same trick as for Accuracy. Let's set `num_examples` as a scalar = 0, instead of torch tensor.